### DISCUSSION TITLE
Can I run this on a VM / virtual windows install?

### DISCUSSION THREAD

**Discussion by jnz86** [simeononsecurity/Windows-Optimize-Harden-Debloat#90] [Q&A] [answered]
I'm on linux desktop and running a Windows virtual machine with Virt-Manager (or VirtualBox). Can I still run this or are the hardware requirements going to be an issue? I could find no talking about the same process.

**Comment by simeononsecurity** [comment, marked as answer]
@jnz86 
Yes, you should test this in a vm first and there are no issues with running it in a vm.
just know if your vm setup doesn't support the hardware requirements you'll miss out on those security features.
